src/routes/channel.js

```diff
@@ -51,29 +51,61 @@ router.post(
 
 /**
  * @route GET api/channels
- * @access Public
+ * @access Private
  * @desc Get all channels
  */
 router.get('/', async (req, res) => {
   try {
-    let allChannels = await Channel.find();
+    await Channel.find()
+      .populate('users')
+      .exec((err, channels) => {
+        channels.map(channel => {
+          return {
+            name: channel.name,
+            description: channel.description,
+            dateCreated: channel.date,
+            id: channel._id,
+            users: channel.users,
```

Comment:
This is also returning the password, this would be a big no

Reply:
@rusye This is because of this part here? We should change this?

`/**
 * @route POST api/users
 * @access Public
 * @desc Register users
 */
router.post(
  '/register',
  [
    check('name', 'Name is required')
      .not()
      .isEmpty(),
    check('email', 'Please add a valid email').isEmail(),
    check('password', 'Please add a password with 6 or more characters').isLength({
      min: 6,
    }),
    //We shouldn't be sending back the attempted password I think`

Reply:
Oh no not that part up top, for some reason all your logic inside the `.exec()` doesn't work, it doesn't send back the data as you specify it and it sends back the version stuff.  Going to share a screenshot in just a second.